### ISSUE TITLE
Keyboard navigation doesn't work

### ISSUE THREAD

**Issue by tvolkert** [flutter/flutter#15756]
- [ ] navigate using arrows
- [ ] navigate using tab
- [ ] navigate using enter
- [ ] escape should not quit the app
- [x] Text fields work with keyboard (this currently works)

**Comment by tvolkert** [comment]
Closing as dup of https://github.com/flutter/flutter/issues/13264
